
From: William Lee Irwin III <wli@holomorphy.com>

Nuke the real undefined symbol in sparc32.  This is the only real hit from
ldchk on sparc32; the rest are all btfixup-related (Sam Ravnborg and I are
working on addressing that).

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/sparc/kernel/entry.S |    2 +-
 25-akpm/arch/sparc/mm/sun4c.c     |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff -puN arch/sparc/kernel/entry.S~sparc-remove-undefined-symbol arch/sparc/kernel/entry.S
--- 25/arch/sparc/kernel/entry.S~sparc-remove-undefined-symbol	Mon Aug  2 15:58:25 2004
+++ 25-akpm/arch/sparc/kernel/entry.S	Mon Aug  2 15:58:25 2004
@@ -858,7 +858,7 @@ vac_hwflush_patch1_on:		addcc	%l7, -PAGE
 vac_hwflush_patch2_on:		sta	%g0, [%l3 + %l7] ASI_HWFLUSHSEG
 
 	.globl	invalid_segment_patch1, invalid_segment_patch2
-	.globl	num_context_patch1, num_context_patch2
+	.globl	num_context_patch1
 	.globl	vac_linesize_patch, vac_hwflush_patch1
 	.globl	vac_hwflush_patch2
 
diff -puN arch/sparc/mm/sun4c.c~sparc-remove-undefined-symbol arch/sparc/mm/sun4c.c
--- 25/arch/sparc/mm/sun4c.c~sparc-remove-undefined-symbol	Mon Aug  2 15:58:25 2004
+++ 25-akpm/arch/sparc/mm/sun4c.c	Mon Aug  2 15:58:25 2004
@@ -379,7 +379,7 @@ extern unsigned long invalid_segment_pat
 extern unsigned long invalid_segment_patch2, invalid_segment_patch2_ff;
 extern unsigned long invalid_segment_patch1_1ff, invalid_segment_patch2_1ff;
 extern unsigned long num_context_patch1, num_context_patch1_16;
-extern unsigned long num_context_patch2, num_context_patch2_16;
+extern unsigned long num_context_patch2_16;
 extern unsigned long vac_linesize_patch, vac_linesize_patch_32;
 extern unsigned long vac_hwflush_patch1, vac_hwflush_patch1_on;
 extern unsigned long vac_hwflush_patch2, vac_hwflush_patch2_on;
_
